gulpfile.js

```diff
@@ -429,49 +429,61 @@ function install(done) {
     // combine dependencies and napa sources into one object
     const dependencies = packages.dependencies;
     for (let key in packages.napa) {
-        if (packages.napa.hasOwnProperty(key)) {
```

Comment:
I'm assuming you've tested the asset builder to verify the linter didn't break anything?